SPEC.md

```diff
@@ -211,6 +211,65 @@ Plugins may define additional fields that they accept and may generate an error
 }
 ```
 
+### Network Configuration Lists
+
+Network configuration lists provide a mechanism to run multiple CNI plugins for a single container in a defined order, passing the result of each plugin to the next plugin.
```

Comment:
In a world where Kubernetes has a Network object, it seems that object would translate to CNI plugins.  Do we expect the user who creates the Network block to be forced to specify both portmapping *and* 'real" driver?  That would force end-users to know about CNI.  Or do we expect them to only specify the real driver, forcing kubelet or dockershim to add the portmapping plugin?  How does kubelet know that the driver specified by th euser doesn't do it natively?